lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -463,6 +463,15 @@ def add_creditcard(xml, creditcard, currency=nil)
         end
       end
 
+      def add_cdpt_authentication_value(xml, creditcard)
+        xml.tag! :AuthenticationECIInd, nil
```

Comment:
@curiousepic if a value is sent, will it cause transactions to fail? Seems like we should send something along if we do have one.